doc/kazoo/cluster-guide.md

```diff
@@ -1,9 +1,275 @@
+```
+
+### Assign Freeswitch to Ecallmgr zones
+Edit the database using Futon or Fauxton by browsing to the following link.  
+http://bc1.z100.somedomain.com:5984/_utils/document.html?system_config/ecallmgr
+
+Add the following to the root of the document.  So at the same level as `"default":`
+```
+"z100": {
+       "fs_nodes": [
+           "freeswitch@fs1.z100.somedomain.com"
+       ]
+},
+"z200": {
+       "fs_nodes": [
+           "freeswitch@fs1.z200.somedomain.com"
+       ]
+   },
+```
+Remove any other `"fs_nodes":[...]` entries in that document.
+
+Refresh to get above changes  
+`sup kapps_maintenance refresh "system_config"`  
+
+Verify on kazoo in each zone that only freeswitch for that zone is visible.    
+`sup ecallmgr_maintenance list_fs_nodes`
+
+Our example cluster assumes ecallmgr is started as a kazoo app on the kazoo server.  If it is started separately, with systemd or init, on the kazoo server or on its own server, the command would be:  
+`sup -necallmgr ecallmgr_maintenance list_fs_nodes`
+
+### Kamailio Config
+
+Each Kamailio configuration at `/etc/kazoo/kamailio/local.cfg` needs to be configured with it's hostname, IP address, and all RabbitMQ servers.  The following config would be for the `ka1.z100` server.
+```
+## CHANGE "" TO YOUR SERVERS HOSTNAME
+#!substdef "!MY_HOSTNAME!ka1.z100.somedomain.com!g"
+
+## CHANGE "127.0.0.1" TO YOUR SERVERS IP ADDRESS
+##     Usually your public IP.  If you need
+##     to listen on addtional ports or IPs
+##     add them in "BINDINGS" at the bottom.
+#!substdef "!MY_IP_ADDRESS!10.100.50.1!g"
+
+## CHANGE "kazoo://guest:guest@127.0.0.1:5672" TO THE AMQP URL
+##     This should be the primary RabbitMQ server 
+##     in the zone that this server will service.
+#!substdef "!MY_AMQP_URL!kazoo://guest:guest@10.100.30.1:5672!g"
+
+## CHANGE "kazoo://guest:guest@127.0.0.1:5672" TO THE AMQP URL for the other zone.
+##     This uses the existing MY_AMQP_SECONDARY_URL variable defined in default.cfg
+##     Note the addition of the "zone=" part in the middle 
+#!substdef "!MY_AMQP_SECONDARY_URL!zone=z200;kazoo://guest:guest@10.200.30.1:5672!g"
+```
+To view the entire cluster and zone setup enter the following on either kazoo server.  
+```kazoo-applications status```
+
+### Kamailio Dispatcher
+Add all Freeswitch servers to the dispatcher configuration on each Kamailio server.  The following example is run on `ka1.z100` kamailio server.  Local zone freeswitch is given setid value of 1, Freeswitch servers in other zones are given setid value of 2.
+```
+sqlite3 /etc/kazoo/kamailio/db/kazoo.db "INSERT INTO dispatcher (setid, destination, flags, priority, attrs, description) \
+VALUES ('1', 'SIP:10.100.20.1:11000', '0', '0', '', 'zone 100')"
+
+sqlite3 /etc/kazoo/kamailio/db/kazoo.db "INSERT INTO dispatcher (setid, destination, flags, priority, attrs, description) \
+VALUES ('2', 'SIP:10.200.20.1:11000', '0', '0', '', 'zone 200')"
+```
+Load the changes
+`kamcmd dispatcher.reload`
+
```

Comment:
the above `Kamailio Dispatcher` is not needed as dispatcher gets automatically populated.
if you add a freeswitch to a zone, it will auto populated dispatcher and reload if necessary